html-css/week1/Ali Sharifi/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+<body class="container">
+    <!-- NavBar -->
+    <nav class="navbar">
+        <ul>
+            <li><a href="#header">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#skills">Skills</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+    </nav>
+
+    <!-- header -->
+    <header id="header">
+        <div class="background">
+            <img  class="avatar" src="avatar.jpeg" alt="">
+            <h1 class="t-word"><span class="s-word">Front-end Developer</span><br />Ali sharifi</h1>
+          </div>
+    </header>
+
+    <!-- About-->
+    <section id="about">
+        <h3>About Me</h3>
+        <hr>
+        <p>
+            I'm Ali, a software engineer passionate about problem-solving and innovation. My tech journey began with a curiosity for coding languages. With a solid grasp of React.js, I have become proficient in front-end development. I enjoy creating intuitive user interfaces and turning ideas into practical code. With a focus on detail and a commitment to quality, I am always eager to expand my knowledge and push the boundaries of web development. 
+        </p>
+    </section>
+
+    <!-- Skill  -->
+    <section id="skills">
+        <h3>Technical Skills</h3>
+        <hr>
+        <div class="skills-icons">
+            <div class="skill">
+                <i class="fab fa-html5"></i>
+                <p>HTML5</p>
+            </div>
+            <div class="skill">
+                <i class="fab fa-css3-alt"></i>
+                <p>CSS3</p>
+            </div>
+            <div class="skill">
+                <i class="fab fa-js"></i>
+                <p>JavaScript</p>
+            </div>
+            <div class="skill">
+                <i class="fab fa-react"></i>
+                <p>React</p>
+            </div>
+            <div class="skill">
+                <i class="fab fa-git-alt"></i>
+                <p>Git</p>
+            </div>
+        </div>
+    </section>
+
+    <!-- Portfolio -->
+    <section id="portfolio">
+        <h3>Portfolio</h3>
+        <hr>
+        <div class="project">
+            <a href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form" target="_blank">
+                FCC Survey Form
+            </a>
+        </div>
+    </section>
+
+
+    <!-- Contact -->
+    <section id="contact">
+        <h3>Contact Me</h3>
+        <hr>
+        <p>Email: mohammadali.sharifi@rocketmail.com</p>
```

Comment:
You can easily convert the email into an active link with this:
```suggestion
        <p>Email: <a href="mailto:mohammadali.sharifi@rocketmail.com">mohammadali.sharifi@rocketmail.com</a></p>
```